deploy_tests/cgap/cgap.py

```diff
@@ -0,0 +1,65 @@
+import json
+import random
+import requests
+from requests.auth import HTTPBasicAuth
+from locust import HttpUser, task, between
+from lib.locust_api import LocustAuthHandler
+from deploy_tests.utils import build_url
+
+
+HOST = 'https://cgap.hms.harvard.edu'
+
+
+# The following item types give no search result and thus 404, so we will not access their collection pages
+BAD_ITEM_TYPES = ['AnnotationField', 'GeneAnnotationField', 'Image', 'QualityMetricPeddyqc', 'TrackingItem',
+                  'WorkflowMapping', 'WorkflowRun']
+
+
+# Configuration
+# 2 User configurations with possible 3rd extension later
+#   * BasicUser - randomly navigate cases
+#   * SearchUser - randomly execute searches
+#   * NavigationUser - TODO: randomly navigate pages when we have more of them
+
+
+class BasicUser(HttpUser):
+    """ Locust user who will randomly get a case, spending 5-10 seconds on the page """
+    host = HOST
+    weight = 1
+    wait_time = between(5, 10)
+    _auth = HTTPBasicAuth(*LocustAuthHandler(is_ff=False).get_username_and_password())  # get CGAP auth
+    cases = list(c['@id'] for c in requests.get(build_url(host, "/Case"), auth=_auth).json()['@graph'])
+    item_types = list(t for t in requests.get(build_url(host, "/counts?format=json")).json()['db_es_compare'].keys()
+                      if t not in BAD_ITEM_TYPES)
+
+    @task(1)
+    def case(self):
```

Comment:
I think this method name needs a verb or somehow a longer name. Really I might think this about all the task names. They look funny as method names.